website/src/pages/index.tsx

```diff
@@ -365,7 +365,7 @@ function EnterpriseReady() {
                 <h2 className="text-gray-900 dark:text-gray-100 text-lg title-font font-medium mb-2">Code signing</h2>
 
                 <p className="leading-relaxed text-base">
-                  macOS binaries are digitally signed (Windows certification is in)
+                  Digitally signed binaries available across Windows and macOS.
```

Comment:
```suggestion
                  Signed binaries for both macOS and Windows
```

And maybe prefer macOS first because usually dev's are: #1 Mac or Linux and then Windows